packages/contracts-bedrock/src/L2/GasPriceOracle.sol

```diff
@@ -93,7 +162,23 @@ contract GasPriceOracle is ISemver {
                 total += 16;
             }
         }
-        uint256 unsigned = total + overhead();
+        uint256 unsigned = total + L1Block(Predeploys.L1_BLOCK_ATTRIBUTES).l1FeeOverhead();
```

Comment:
```suggestion
        uint256 unsigned = total + isEcotone ? 0 : L1Block(Predeploys.L1_BLOCK_ATTRIBUTES).l1FeeOverhead();
```

...From Slack discussion where the proposal is to just make this return "calldataGas".

then you can drop the _getL1GasUsedEcotone() function entirely.

Reply:
worth noting that with this approach the _getL1FeeBedrock / _getL1FeeEcotone methods are checking the `isEcotone` flag twice. are we ok with that additional gas usage?